maybe/index.ts

```diff
@@ -29,7 +30,7 @@ type StaticCheck = ClassImplements<
   [MonadConstructor, ApplicativeConstructor, AsyncChainable<Maybe<any>>]
 >;
 export default class MaybeConstructor<T, S extends MaybeState = MaybeState>
-  implements AsyncMonad<T>, Alternative<T>, Container<T> {
+  implements AsyncMonad<T>, Alternative<T>, Container<T>, Catamorphism<[undefined, T]> {
```

Comment:
Put here `void` please, to eliminate argument in `mapNone`

Reply:
Fixed